### ISSUE TITLE
Template UWP project doesn't initially load the NuGet package

### ISSUE THREAD

**Issue by maluoi** [maluoi/StereoKit#177]
When creating a UWP project, intellisense doesn't load until the first build. The NuGet package isn't getting downloaded/loaded on project creation, instead it waits until the first build before resolving the package. I _think_ there should be a way to make initial NuGet package resolution happen a little earlier.

**Comment by cwule** [comment]
I always use "Restore NuGet packages", which also does the trick.

**Comment by maluoi** [comment]
Yeah, that sounds right! I believe building/running the project will also trigger a restore action as well?

This one isn't a blocker, but it is a polish detail that I would _love_ to have working if possible! First impressions and all that :D

**Comment by cwule** [comment]
Yes, always looks a bit scary with everything red :)
